source/code-generation/query-converter/convert-stored-procedures.txt

```diff
@@ -10,3 +10,276 @@ Convert Stored Procedures
    :depth: 1
    :class: singlecol
 
+You can convert SQL stored procedures to MongoDB syntax with the query
+converter. The query converter considers the mapping rules and schema
+transformations defined in your project when converting your SQL code.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+Before you Begin
+----------------
+
+.. include:: /includes/fact-query-converter-disclaimer.rst
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Navigate to the query converter pane
+
+  	   From the :guilabel:`Code Generation` tab, click the :guilabel:`Query Converter` pane.
+
+   .. step:: Open the query converter view
+
+      -  If it is your first time using the query converter in your
+         project, click :guilabel:`Import From Database`.
+
+      -  If your project already has converted SQL code, click the
+         :guilabel:`Manage Database Objects` button on the left-hand
+         pane.
+
+   .. step:: Connect to your relational database
+
+      a. Specify the connection details to your source database. 
+         
+         For details, see :ref:`rm-relational-database-connection-strings`.
+      #. Click :guilabel:`Connect`.
+
+   .. step:: Select stored procedures
+
+      a. On the :guilabel:`Import Database Objects` modal, click
+         the :icon-fa5:`chevron-right` icon next :guilabel:`Database`.
+
+      #. Click the :icon-fa5:`chevron-right` icon next to your schema.
+
+      #. Click the :icon-fa5:`chevron-right` icon next to
+         :guilabel:`Stored Procedures`.
+
+         .. tip::
+
+            You can toggle stored procedures for conversion by clicking
+            the :icon-fa5:`check-square` icon next to a stored procedure's
+            name. All stored procedures are selected by default.
+
+      #. Click :guilabel:`Save`.
+
+         The SQL syntax for each stored procedure in your database 
+         schema is imported into your project and is visible in the 
+         :guilabel:`Query Converter` pane under 
+         :guilabel:`Stored Procedures`.
+
+   .. step:: Convert and test syntax
+
+      a. Click a stored procedure's name from the left-hand pane under
+         :guilabel:`Stored Procedures`.
+         
+         The SQL syntax displays in the 
+         :guilabel:`Imported Stored Procedure` pane.
+
+      #. Select a :guilabel:`Target Language` from the drop-down.
+         
+         Query converter supports stored procedure conversions to
+         ``Javascript``, ``C#``, and ``Java``.
+
+      #. Click the :guilabel:`Convert` button. Wait for the query 
```

Comment:
Not sure if we want to go into details about what is in and out of scope for conversions here or just remain procedural. I did get the following error when attempting to convert [a SP which referenced a temporary table ](https://gist.github.com/ianf-mongodb/da452c525389d142bec137abae78bb51):

<img width="1394" alt="Screenshot 2023-11-07 at 11 28 32 AM" src="https://github.com/mongodb/docs-relational-migrator/assets/85948430/8eb9c05d-d20a-42ce-b88a-25e620340f9a">

Reply:
I think this information would go in **About this Task** if we decide to include it.

Reply:
@ianf-mongodb Some of these error cases will be temporary as we are still making improvements. Probably no need to focus on the exact cases that do/don't work in the docs as they will change.